ui/prometheus-plugin/src/plugins/prometheus-datasource.ts

```diff
@@ -33,6 +33,7 @@ const createClient: DatasourcePlugin<PrometheusDatasourceSpec, PrometheusClient>
 
   // Could think about this becoming a class, although it definitely doesn't have to be
   return {
+    options: options,
```

Comment:
I think you'll want to follow the same URL logic as above when exposing this to autocomplete. Basically `proxyUrl` is for datasources that want the Perses backend to act as a proxy (e.g. to Prometheus) while `direct_url` on the spec is for clients that want to talk directly to an instance from the browser without a proxy in-between.

For our internal use, I think we'll always have a `direct_url` not a proxy one.

Reply:
Ohh I see. yeah good call, I'll fix that up